compiler/damlc/daml-compiler/src/DA/Daml/Compiler/Dar.hs

```diff
@@ -138,8 +143,8 @@ mkConfFile PackageConfigFields {..} pkgModuleNames pkgId = (confName, bs)
         BSC.pack $
         unlines
             [ "name: " ++ pName
-            , "id: " ++ pName -- will change to key
-            , "key: " ++ pName -- will change to key
+            , "id: " ++ key
```

Comment:
Does this change mean that users will have to run `daml clean` if they had run `daml init` (via `daml build`) before? If so, we should mention it in the release notes.